pom.xml

```diff
@@ -11,7 +11,7 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>rgraph</artifactId>
-    <version>3_2015-07-27-stable-SNAPSHOT</version>
+    <version>RGraph4.66-stable</version>
```

Comment:
This should be `4.66-stable-SNAPSHOT`

Reply:
Do you want me to made the change in the version number?